pubspec.yaml

```diff
@@ -3,25 +3,24 @@
 # BSD-style license that can be found in the LICENSE file.
 
 name: ffigen
-version: 1.2.0
+version: 2.0.0-nullsafety
 homepage: https://github.com/dart-lang/ffigen
 description: Experimental generator for FFI bindings, using LibClang to parse C header files.
 
 environment:
-  sdk: '>=2.7.0 <3.0.0'
+  sdk: '>=2.12.0-29.10.beta <3.0.0'
 
 dependencies:
-  ffi: ^0.1.3
-  yaml: ^2.2.1
-  meta: ^1.1.8
-  args: ^1.6.0
-  logging: ^0.11.4
-  cli_util: ^0.2.0
-  glob: ^1.2.0
-  path: ^1.7.0
-  quiver: ^2.1.3
-  pub_semver: ^1.4.4
+  ffi: ^0.2.0-nullsafety.1
+  yaml: ^3.0.0-nullsafety.0
+  path: ^1.8.0-nullsafety.3
+  quiver: ^3.0.0-nullsafety.2
+  args: ^1.6.0        # Not available
```

Comment:
Please note these in the GitHub issue which mentions that we do unsound null safety for now.

(Note that unsound null safety is fine, because people only use this package as a dev dependency!)

Reply:
Done - #128